security/address-bar-spoofing/spoof-form-action.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <meta http-equiv="Content-Security-Policy" content="form-action https://duck.co" />
+    <title>URL Spoofing - Redirect Form Action</title>
+    <script>
+        // eslint-disable-next-line no-unused-vars
+        function run() {
```

Comment:
there is nothing triggering allowing to trigger "run" on this page, is that by design?